src/shared/schema/us/actions/collections.ts

```diff
@@ -59,8 +59,7 @@ export const collectionsActions = {
                 collectionId,
                 includePermissionsInfo,
                 filterString,
```

Comment:
Let's rename the action to `getStructureItems`, renaming is not necessary for actions in Redux, but here it is better to do because it will be obvious in Network tab what method is called in us.